modules/exllamav2.py

```diff
@@ -61,8 +61,8 @@ def from_pretrained(self, path_to_model):
 
         if shared.args.cache_8bit:
             cache = ExLlamaV2Cache_8bit(model, lazy=shared.args.autosplit)
-        elif shared.args.cache_4bit:
-            cache = ExLlamaV2Cache_Q4(model, lazy=shared.args.autosplit)
+        elif share.args.cache_q4:
+            cache = ExLlamaV2Cache_Q4(self.ex_model)
```

Comment:
There was an issue that got solved with `lazy=shared.args.autosplit` when I added this flag, so this shouldn't be changed.

Reply:
oops this is accidental change, will revert